### ISSUE TITLE
Editor tracking doesn´t work through vscode WSL instance

### ISSUE THREAD

**Issue by RajceP** [swdotcom/swdc-vscode#87]
**Describe the bug**
```paste below
When i run vscode through WSL, only tracking that works is Code Time.
Active code time, Lines added, Lines removed etc. stays at zero.
```
**ISSUE TYPE**
- Bug Report

**Steps to reproduce**

1. Run vscode from WSL with WSL Remote extension.
2. Code for some time.
3. Check metrics.
4. See zeros.

**Expected behavior**
```paste below
Editor based metrics are tracked.
```

**ACTUAL RESULTS**

```paste below
Only Code time is tracked (not editor based).
```

**Desktop:**
 - OS: Windows 10 1909, WSL1, Ubuntu 18.04 LTS
 - Plugin version: v2.2.3

**Comment by FreekVR** [comment]
Also having this issue, on WSL2, so may be more related to the remote extension rather than WSL itself, but that's just guesswork on my part ;)

**Comment by RajceP** [comment]
Forcing extension to run in workspace instead of UI helped.

settings.json:

`  "remote.extensionKind": {
    "softwaredotcom.swdc-vscode": "workspace"
  }`

**Comment by gregkonush** [comment]
@RajceP adding the above settings to vscode doesn't yield any results for me. Everything is still at zero.

**Comment by Fyko** [comment]
[RacejP's workaround](https://github.com/swdotcom/swdc-vscode/issues/87#issuecomment-629122813) seems to work for me but WSL crashes when trying to login

**Comment by xavluiz** [comment]
We apologize for the late response. We're looking into this now and thank you for the information.

**Comment by xavluiz** [comment]
@Fyko @RajceP @gregkonush, we've published v2.3.6 to fix code time and editor tracking logic. The issue was that we filtered out documents that didn't match the "file" or "untitled" scheme, but using WSL tags documents with the "vscode-remote" scheme. Let us know if there's anything else, we appreciate your feedback.

**Comment by xavluiz** [comment]
Please let us know if there are any other issues after upgrading to v2.3.6 or later, thank you.
